### ISSUE TITLE
Separated complex in simulation results

### ISSUE THREAD

**Issue by ivandon15** [tdudgeon/simple-simulate-complex#6]
Hi, 

Have you ever encountered a situation in your simulation results where the protein's position remains roughly unchanged, but the ligand jumps from one side to the other (seemingly influenced by boundary conditions)? 
<img width="819" alt="image" src="https://github.com/tdudgeon/simple-simulate-complex/assets/45418272/12d1327f-6f8d-4736-8943-235e242542cd">

Besides that, there have also been instances where the ligand and the protein are very far apart, appearing to be unrelated to each other. (Figures depicted as below)
<img width="303" alt="image" src="https://github.com/tdudgeon/simple-simulate-complex/assets/45418272/4969599a-91ee-4f49-8b6e-b377304b825b">
<img width="339" alt="image" src="https://github.com/tdudgeon/simple-simulate-complex/assets/45418272/e5d162cb-5441-4b11-bcc4-7600424e1b63">

This really confuses me, and I also tried  to set `enforcePeriodicBox=True` when outputting the traj.

Thanks for your contributions! It really helps me a lot.

**Comment by tdudgeon** [comment]
Yes, this looks like a periodic boundary issue. Did you try using the `analyse.py` script to re-image the trajectory?

**Comment by ivandon15** [comment]
Thank you for replying!
Yes, I do use `analyse.py` to reimage. And as mentioned before, I also tried `enforcePeriodicBox=True`, but it seems that none of them work.
Besides, I think probably re-imaging is only changing the position of the entire complex but remain the relative position unchanged, right?
